vividus-plugin-azure-functions/src/main/java/org/vividus/azure/functions/steps/FunctionSteps.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright 2021 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.vividus.azure.functions.steps;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import org.apache.commons.lang3.StringUtils;
+import org.jbehave.core.annotations.When;
+import org.vividus.azure.functions.service.FunctionService;
+import org.vividus.bdd.context.IBddVariableContext;
+import org.vividus.bdd.variable.VariableScope;
+import org.vividus.util.json.JsonUtils;
+
+import reactor.core.publisher.Mono;
+
+public class FunctionSteps
+{
+    private final JsonUtils jsonUtils;
+    private final FunctionService functionsService;
+    private final IBddVariableContext bddVariableContext;
+
+    public FunctionSteps(JsonUtils jsonUtils, FunctionService functionsService, IBddVariableContext bddVariableContext)
+    {
+        this.jsonUtils = jsonUtils;
+        this.functionsService = functionsService;
+        this.bddVariableContext = bddVariableContext;
+    }
+
+    /**
+     * Triggers Azure function by its name. And saves responses from the service into variable with
+     *  a provided scope and name
+     * @param functionApp   the name of Azure Function App. The value can be retrieved by looking
```

Comment:
Capital letter for description of parameters ? (as https://github.com/vividus-framework/vividus/blob/0943965b8db6c321acf89223946ff1b3a42af7fb/vividus-plugin-azure-functions/src/main/java/org/vividus/azure/functions/steps/FunctionSteps.java#L56)